tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
```

Comment:
Any way to have grunt read from here rather than defining these options in multiple places?